common/src/main/java/com/graphicsfuzz/common/util/PipelineInfo.java

```diff
@@ -402,7 +402,34 @@ private JsonObject lookupUniform(String uniformName) {
 
   @Override
   public PipelineInfo clone() {
-    return new PipelineInfo(toString());
+
+    final PipelineInfo pipelineinfo = new PipelineInfo(toString());
```

Comment:
I suggest calling this `result` - I like to follow a convention where the thing to ultimately be returned is called `result`.  (I wouldn't suggest this minor change if I wasn't requesting other changes :))